libs/plugins/stream-client/src/lib/core/state/clean-dangling-tasks-output-mappings.ts

```diff
@@ -1,43 +1,40 @@
 import { isEmpty, pick, fromPairs } from 'lodash';
-import { ContributionSchema, isMapperActivity } from '@flogo-web/core';
 import { Dictionary } from '@flogo-web/lib-client/core';
 
 import { Item } from '../interfaces';
 import { FlogoStreamState } from './stream.state';
+import { ROOT_TYPES } from '../../shared/mapper/constants';
```

Comment:
we shouldn't have core reference other modules, modules should reference core instead. This is to avoid circular dependencies. We can move the root types to core instead.